Formula/s/slepc-complex.rb

```diff
@@ -0,0 +1,40 @@
+class SlepcComplex < Formula
+  desc "Scalable Library for Eigenvalue Problem Computations (complex)"
+  homepage "https://slepc.upv.es"
+  url "https://slepc.upv.es/download/distrib/slepc-3.22.2.tar.gz"
+  sha256 "b60e58b2fa5eb7db05ce5e3a585811b43b1cc7cf89c32266e37b05f0cefd8899"
+  license "BSD-2-Clause"
+
+  livecheck do
+    formula "slepc"
+  end
+
+  depends_on "petsc-complex"
+
+  uses_from_macos "python" => :build
+
+  conflicts_with "slepc", because: "slepc must be installed with either real or complex support, not both"
+
+  def install
+    ENV["PETSC_DIR"] = Formula["petsc-complex"].prefix.realpath
+    ENV["SLEPC_DIR"] = Dir.pwd
+    system "./configure", "--prefix=#{prefix}"
```

Comment:
Does this work?
```suggestion
    system "./configure", *std_configure_args
```

Reply:
looks like it does not work, added some notes about it

Reply:
What is the fatal error? Is it in brew? Upstream?

Reply:
it would print like this

```
==> ./configure --disable-debug --disable-dependency-tracking --prefix=/opt/homebrew/Cellar/slepc-complex/3.22.2 --libdir=/opt/homebrew/Cellar/slepc-complex/3.22.2/lib
Checking environment... ERROR: Invalid arguments --with-debug=0 --with-dependency-tracking=0 --libdir=/opt/homebrew/Cellar/slepc-complex/3.22.2/lib
Use -h for help
```

Reply:
The SLEPc configure is a python script, it does not have autoconf-compatible options